pkg/controller/appdefinition/status.go

```diff
@@ -162,6 +163,46 @@ func JobStatus(req router.Request, resp router.Response) error {
 	return nil
 }
 
+func VolumeStatus(req router.Request, resp router.Response) error {
+	var (
+		app  = req.Object.(*v1.AppInstance)
+		cond = condition.Setter(app, resp, v1.AppInstanceConditionVolumes)
+		pvcs = new(corev1.PersistentVolumeClaimList)
+
+		messages []string
+	)
+
+	if err := req.List(pvcs, &kclient.ListOptions{
+		Namespace: app.Status.Namespace,
+		LabelSelector: klabels.SelectorFromSet(map[string]string{
+			labels.AcornManaged: "true",
+			labels.AcornAppName: app.Name,
```

Comment:
Do we actually need `"acorn.io/volume-name" label? I don't see a place where you use it.